cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/export/CgmesExportUtil.java

```diff
@@ -315,12 +304,27 @@ public static String getTerminalId(Terminal t, CgmesExportContext context) {
         if (c instanceof DanglingLine) {
             aliasType = Conversion.CGMES_PREFIX_ALIAS_PROPERTIES + CgmesNames.TERMINAL1;
         } else {
-            int sequenceNumber = getTerminalSequenceNumber(t);
+            int sequenceNumber = getTerminalSequenceNumber(t, Collections.emptyList()); // never a dangling line here
             aliasType = Conversion.CGMES_PREFIX_ALIAS_PROPERTIES + CgmesNames.TERMINAL + sequenceNumber;
         }
         return context.getNamingStrategy().getCgmesIdFromAlias(c, aliasType);
     }
 
+    public static List<DanglingLine> getUnpairedDanglingLines(Network network) {
```

Comment:
@flo-dup thinks that when exporting a CGM, it does not handle an subnetwork internal tie line. It means that :
````
    public static boolean isUnpaired(Network network, DanglingLine danglingLine) {
        // For this network, an unpaired dangling line has an unpaired status
        // or its parent network different that this network.
        return !danglingLine.isPaired()
                || danglingLine.isPaired() && danglingLine.getTieLine().orElseThrow().getParentNetwork() != network;
    }
````
 must have like a parent > network.